sources/core-iaa/sources/driver_loader/hw_configuration_driver.c

```diff
@@ -191,7 +191,7 @@ bool own_load_configuration_functions(void *driver_instance_ptr) {
 
         if (err_message) {
             // @todo this is a workaround to optionally load accfg_device_get_iaa_cap
-            char *iaa_cap_func_name = "accfg_device_get_iaa_cap";
+            const char *iaa_cap_func_name = "accfg_device_get_iaa_cap";
```

Comment:
I'm curious as to why this line was changed, but not the L197 for instance.

Reply:
That's a fair question! At the moment, the AI coder is not very good at prioritizing fix attempts; I believe this is a direct consequence of that. I'll try to confirm that as we are actively working on improvements to the selection part of the process now.

Reply:
sure, I'll set it to pending state meanwhile.

Reply:
hi @mzhukova , this branch has been regenerated and pushed -- catching the below change and some additional other cases.